provider/resources.go

```diff
@@ -345,8 +345,13 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 	config.SharedConfigFiles = []string{configPath}
 
 	if _, err := awsbase.GetAwsConfig(context.Background(), config); err != nil {
-		return fmt.Errorf("unable to validate AWS credentials " +
-			"- see https://pulumi.io/install/aws.html for details on configuration")
+
+		return fmt.Errorf("unable to validate AWS credentials. Make sure you have: \n\n" +
+			" \t • Configured your AWS access key \n" +
+			" \t • Configured your AWS_ access key ID \n" +
```

Comment:
I think the environment variables in the previous commit were more clear.

Reply:
It's not how everyone sets up their creds, though, and in the docs, we do not recommend this as the best option.